### PR TITLE
init/iolog: add support for reading iolog from stdin

### PR DESCRIPTION
**Pull request by donny372**
Fixes: #997

Signed-off-by: Shundong Zhou <donny372@gmail.com>

### CODE REVIEW

**Comment by donny372** [comment]
@sitsofe, could you please take a look?

File: init.c [hunk 1]
```diff
@@ -2062,6 +2062,16 @@ static int __parse_jobs_ini(struct thread_data *td,
 		}
 
 		ret = fio_options_parse(td, opts, num_opts);
+
+		if (!ret) {
+			if (!strcmp(file, "-") && td->o.read_iolog_file != NULL) {
+				char *fname = get_name_by_idx(td->o.read_iolog_file, td->subjob_number);
```
Comment (sitsofe) [inline init.c:2068]: Maybe wrap this at the command and align the line below to the bracket.

File: init.c [hunk 2]
```diff
@@ -2062,6 +2062,16 @@ static int __parse_jobs_ini(struct thread_data *td,
...
+				if (!strcmp(fname, "-")) {
+					log_err("fio: we can't read both iolog and job file from stdin.\n");
```
Comment (sitsofe) [inline init.c:2070]: Maybe more wrapping here:
```
                                        log_err("fio: we can't read both iolog " 
                                                "and job file from stdin.\n"); 
```

**Comment by sitsofe** [comment]
Ah maybe there's one less minor thing: could you put the area being affected in the title? e.g.

```
init/iolog: add support for reading iolog from stdin
```

File: init.c (hunk 1 above)
Comment (donny372) [reply init.c]: Done.

File: init.c (hunk 2 above)
Comment (donny372) [reply init.c]: Done.

**Comment by donny372** [comment]
> Ah maybe there's one less minor thing: could you put the area being affected in the title? e.g.
> 
> ```
> init/iolog: add support for reading iolog from stdin
> ```

Done.

**Pull request merged**